Automattic/vip-go-mu-plugins
# Site Details: Include plugins that are enabled sitewide

## Description

We are currently fetching the list of enabled plugins with `get_option( 'active_plugins' )`, but this is not taking into account plugins that are "network active" on a multisite. This PR checks also `get_site_option( 'active_sitewide_plugins' )`, and adds an extra flag `enabled_sitewide` to indicate whether a plugin is enabled network wide.

## Changelog Description

### Include sitewide plugins in site details

We keep improving the site details data we are gathering from all sites to have more information about multiple runtime variables. This time, we are including the list of plugins enabled sitewide on multisite installations, as we were only considering plugins enabled locally for a site.

## Checklist


## Steps to Test

1. Check out PR on a Go Sandbox
1. Execute `wp cron-control events list` and find the id for the entry `vip_config_sync_cron`
1. Run it manually with `wp cron-control events run <id>`
1. Check the output in `/chroot/tmp/logstash.log` and ensure the new field `enabled_sitewide` is present
